libs/tools/sharepoint/msal.js

```diff
@@ -0,0 +1,62 @@
+import { loadScript, getConfig } from '../../utils/utils.js';
+import { getSiteConfig, spAccessToken } from './state.js';
+
+let msalConfig;
+
+const login = { redirectUri: '/tools/loc/spauth' };
+const siteKeys = ['clientId', 'authority', 'site', 'root'];
+const cache = {
+  cacheLocation: 'sessionStorage',
+  storeAuthStateInCookie: false,
+};
+
+const telemetry = {
+  application: {
+    appName: 'Adobe Franklin Localization',
+    appVersion: '0.0.1',
+  },
+};
+
+export function getMSALConfig() {
+  // eslint-disable-next-line no-async-promise-executor
+  return new Promise(async (resolve) => {
+    if (!msalConfig) {
+      const { configs = {} } = await getSiteConfig();
+      const { data = [] } = configs;
+      const configValues = {};
+      siteKeys.forEach((key) => {
+        const currentData = data.find((item) => (item.key === `prod.sharepoint.${key}`));
+        configValues[key] = currentData?.value;
+      });
+      const { clientId, authority, site, root } = configValues;
+      const auth = { clientId, authority };
+      const config = getConfig();
+      const base = config.miloLibs || config.codeRoot;
+
+      try {
+        await loadScript(`${base}/deps/msal-browser-2.34.0.js`);
+      } catch (err) {
+        console.log(err);
+      }
+      msalConfig = {
+        login,
+        auth,
+        cache,
+        telemetry,
+        site,
+        baseUri: `${site}/drive/root:/${root}`,
```

Comment:
This might fail for CC as it lives in a separate drive. The drive id value needs to be part of the API call. 
https://github.com/adobecom/milo/blob/main/tools/loc/config.js#L106

Reply:
Fixed.